src/app.js

```diff
@@ -1 +1,8 @@
 'use strict';
+
+const { randomNumber } = require('./randomNumber');
+const { guessNumber } = require('./guessNumber');
+
+const number = randomNumber();
+
+guessNumber(number);
```

Comment:
Consider renaming the 'guessNumber' function to something more descriptive like 'startGame' or 'playGame', as it is not actually guessing the number but rather starting the game.